src/Apps/Order/Routes/__tests__/Shipping.test.tsx

```diff
@@ -37,18 +37,14 @@ describe("Shipping", () => {
 
   it("commits the mutation with the orderId", () => {
     const component = getWrapper(props)
-    component
-      .find(Radio)
-      .last()
-      .simulate("click")
```

Comment:
Was this just completely unrelated?

Reply:
Correct – just copy/pasted from a test by accident.